src/opm/output/eclipse/LoadRestart.cpp

```diff
@@ -75,6 +75,10 @@ class RestartFileView
         return this->sim_step_;
     }
 
+    int reportStep() const {
```

Comment:
I expected this to go away when we merged #800.  Do you need to rebase?

Reply:
Might be